src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -188,6 +197,14 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISc
     /// <summary>
     /// Estimator for <see cref="IidChangePointDetector"/>
     /// </summary>
+    /// <p>Example code can be found by searching for <i>IidChangePointDetector</i> in <a href='https://github.com/dotnet/machinelearning'>ML.NET.</a></p>
+    /// <example>
+    /// <format type="text/markdown">
+    /// <![CDATA[
+    /// [!code-csharp[MF](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs)]
+    /// ]]>
+    /// </format>
+    /// </example>
```

Comment:
This section was moved here, and removed from the constructor below in https://github.com/dotnet/machinelearning/commit/707aad1114111ccb510b258cf1339644eb289d3e.

I believe the correct merge here is to remove lines 220-225 below. (it looks like the `<example>` tags were already removed below.)